var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -628,7 +647,7 @@ def cmake_args(self):
 
         if "+gold" in spec:
             cmake_args.append(
-                define("LLVM_BINUTILS_INCDIR", spec["binutils"].prefix.include)
+                define("LLVM_BINUTILS_INCDIR", self.find_binutils_header_dir())
```

Comment:
It might be more straightforward to use: `self.spec['binutils'].headers.directories`